packages/utils/tests/eslint-utils/getParserServices.test.ts

```diff
@@ -26,6 +26,16 @@ const createMockRuleContext = (
   }) as unknown as UnknownRuleContext;
 
 describe('getParserServices', () => {
+  const requiresParserServicesMessageTemplate =
+    'You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.\n' +
+    'Parser: \\S*';
+  const baseErrorRegex = new RegExp(requiresParserServicesMessageTemplate);
+  const unknownParserErrorRegex = new RegExp(
+    requiresParserServicesMessageTemplate +
+      '\n' +
+      'Note: detected a parser other than @typescript-eslint/parser. Make sure the parser is configured to forward "parserOptions.project" to @typescript-eslint/parser.',
+  );
```

Comment:
[Style] Nit: mocks, test regexes, and the like are generally declared in the root of test files - including this one. For consistency I think it's be best to stick with just one style. (I don't personally have a strong preference for one over the other in isolation otherwise)

Reply:
Thanks. I made the change just now.